src/components/staking/PageValidator.vue

```diff
@@ -2,7 +2,7 @@
   <TmPage
     :managed="true"
     :data-empty="!validator.operatorAddress"
-    :loading="$apollo.loading"
+    :loading="$apollo.queries.validator.loading"
     :loaded="loaded"
```

Comment:
can you make sure loaded is being set properly?

Reply:
I just did. It is being set properly